resources/Schema/Components/ExplosionEffect.xml

```diff
@@ -1,19 +1,21 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no" ?>
 <ExplosionEffect xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="ExplosionEffect.xsd">
 	<ExplosionOrigin X="0" Y="0" Z="0"/>
-	<TimeSinceDeath>0</TimeSinceDeath>
-	<ExplosionDuration>2</ExplosionDuration>
+	<TimeSinceDeath>0.0</TimeSinceDeath>
+	<ExplosionDuration>2.0</ExplosionDuration>
 	<Speed>1</Speed>
 	<Delay>0</Delay>
 	<!--<Gravity>1</Gravity>-->
 	<!--<GravityForce>1</GravityForce>-->
 	<!--<ObjectRadius>2</ObjectRadius>-->
 	<EndColor R="0" G="0" B="0" A="0"/>
-	<Randomness>0</Randomness>
-	<RandomnessScalar>1</RandomnessScalar>
+	<Randomness>false</Randomness>
+	<RandomnessScalar>1.0</RandomnessScalar>
 	<Velocity X="2" Y="2"/>
-	<ColorByDistance>0</ColorByDistance>
-	<!--<ReverseAnimation>0</ReverseAnimation>-->
+	<ColorByDistance>false</ColorByDistance>
 	<!--<Wireframe>0</Wireframe>-->
-	<ExponentialAccelaration>0</ExponentialAccelaration>
+	<ExponentialAccelaration>false</ExponentialAccelaration>
+	<Pulsate>false</Pulsate>
+	<Reverse>false</Reverse>
```

Comment:
I added the ``Speed`` field to the component which just runs the effect time backwards if you pass -1 instead of 1. Do we still need "Reverse"?